### PR TITLE
Fixes leave organisation button error

### LINKED ISSUE #176: Leave Organization button edge case error
If user is not a part of any organization then a `snackbar` should be created informing, `user is not a part of any organisation`. Currently, Alert Dialogue is displayed asking for the permission to leave even if user is added to no organisation at all.

WhatsApp Image 2021-03-14 at 8 41 21 AM

### PR DESCRIPTION
**Pull request by SriramPatibanda**
**What kind of change does this PR introduce?**

BugFix

**Did you add tests for your changes?**

No

**If relevant, did you update the documentation?**

No

**Summary**

Updates the app such that when a user isn't a part of any organization, a snackbar with a warning message is displayed. Adds joinedOrgs variable and updates FlatButton to TextButton at places as FlatButton is now deprecated.

Fixes #176 

**Does this PR introduce a breaking change?**

No

**Other information**

N.A.

### CODE REVIEW

**Comment by SriramPatibanda** [comment]
@Sagar2366 Please review this PR whenever possible

**Comment by Sagar2366** [comment]
If user is not part of any organization then remove leave organization option for the user or disable it.

**Comment by SriramPatibanda** [comment]
> If user is not part of any organization then remove leave organization option for the user or disable it.

Yeah, I was thinking of proposing but did this anyway. Will make the necessary changes soon.

**Comment by SriramPatibanda** [comment]
@Sagar2366 The requested changes have been made, kindly review them whenever possible. I also removed the variable that I added in my previous commit i.e. joinedOrgs as I understood that it was unnecessary.

**Comment by SriramPatibanda** [comment]
@Sagar2366 along with this, I think even the switch organization button can be made such that it's shown only when the user in more than 1 organisations. What do you think? Should I cover even that in this PR or do you want me to make a separate PR if you think that this change is necessary?

**Comment by Sagar2366** [comment]
@SriramPatibanda pull latest changes from master, this feature is already added.
In recent changes this functionality is also added - if user is part of multiple organization and leaves current organization then next organization from joined group will be automatically be considered as current organization. But as you are on older code, it is showing null, and also not removing leave organization button. Please test this scenario again on your PR after pulling master changes.

**Comment by SriramPatibanda** [comment]
@Sagar2366 I've pulled the latest changes and there are some errors in the code. If you observe, the register_form.dart file has commented code at the bottom of the file. The lastest PR that has been merged seems to have some issues.

**Comment by SriramPatibanda** [comment]
@Sagar2366 I've fixed the error and can push the changes to this repo if you permit me to.

**Comment by SriramPatibanda** [comment]
> @SriramPatibanda pull latest changes from master, this feature is already added.
> In recent changes this functionality is also added - if user is part of multiple organization and leaves current organization then next organization from joined group will be automatically be considered as current organization. But as you are on older code, it is showing null, and also not removing leave organization button. Please test this scenario again on your PR after pulling master changes.

Yeah the changes you mentioned are indeed there, but what I suggested was something a little different. Whenever a user is there in only one organisation, there's technically no need to show the switch organization button as he's there in only 1 org. So I suggest that show the switch org button only when the user in 2 or more orgs.  Let me know what you think about it.

**Comment by SriramPatibanda** [comment]
I've fixed the errors in the register form and the code is ready to be reviewed and merged.

File: lib/views/pages/login_signup/register_form.dart [hunk 1]
```diff
@@ -471,7 +465,8 @@ class RegisterFormState extends State<RegisterForm> {
           );
         });
   }
-  /*_successToast(String msg) {
+
```
Comment (Sagar2366) [inline lib/views/pages/login_signup/register_form.dart:468]: This changes are already added in PR https://github.com/PalisadoesFoundation/talawa/pull/373

Comment (SriramPatibanda) [reply lib/views/pages/login_signup/register_form.dart:468]: When I made the PR, they weren't and that's why I did them. No problem, I'll just revert these changes, pull and push.

Comment (SriramPatibanda) [reply lib/views/pages/login_signup/register_form.dart]: @Sagar2366 , I've made the required changes. Kindly review them.

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**